presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py

```diff
@@ -0,0 +1,62 @@
+from PIL import Image, ImageChops
+from presidio_image_redactor.image_analyzer_engine import ImageAnalyzerEngine
+import matplotlib
+import io
+from matplotlib import pyplot as plt
+
+
+def fig2img(fig):
+    """Convert a Matplotlib figure to a PIL Image and return it."""
+
+    buf = io.BytesIO()
+    fig.savefig(buf)
+    buf.seek(0)
+    img = Image.open(buf)
+    return img
+
+
+class ImagePiiVerifyEngine:
+    """ImagePiiVerifyEngine class only supporting Pii verification currently."""
+
+    def __init__(self):
+        self.analyzer_engine = ImageAnalyzerEngine()
+
+    def verify(self, image: Image) -> Image:
+        """Redact method to redact the given image.
```

Comment:
```suggestion
        """Annotate image with the detect PII entity.
```
Or something similar :)